gatewayconfig/gatewayconfig_app.py

```diff
@@ -13,10 +13,11 @@
 from gatewayconfig.processors.bluetooth_advertisement_processor import BluetoothAdvertisementProcessor
 from gatewayconfig.gatewayconfig_shared_state import GatewayconfigSharedState
 from gatewayconfig.file_loader import read_eth0_mac_address, read_wlan0_mac_address
-from gatewayconfig.gpio.mraa_button import MraaButton
-from gatewayconfig.gpio.mraa_led import MraaLED
 import gatewayconfig.nmcli_custom as nmcli_custom
 
+if is_rockpi():
```

Comment:
this gets executed when this file is imported. That is why the test had to be modified. this import can be moved to where it is used, that will be cleaner IMHO.